LayoutKitSampleApp/Benchmarks/CollectionViewController.swift

```diff
@@ -8,6 +8,23 @@
 
 import UIKit
 
+
+/// Each view type of collection view should subclass from `CollectionViewController`
+/// If each view type uses `CollectionViewController`, it will cause the crash.
```

Comment:
I'd add these two bits of text to this comment to stress what's important about these derived classes:

>Each view type of collection view should *be a non-generic* subclass from `CollectionViewController`
If each view type uses *the generic* `CollectionViewController`, it will cause the crash.